### PR TITLE
creating product, contact, about pages

### PR DESCRIPTION
**Pull request by Anas-Ayoub**

### CODE REVIEW

File: ecommerce/urls.py [hunk 1]
```diff
@@ -3,5 +3,10 @@
 
 urlpatterns = [
     path('',views.Home.as_view(),name='home'),
+    path('products/', views.Products.as_view(),'products'),
```
Comment (yasser-sab) [inline ecommerce/urls.py]: i think you forget to put the name attribute on third argument

File: ecommerce/urls.py [hunk 2]
```diff
@@ -3,5 +3,10 @@
...
+    path('single_product/', views.SingleProduct.as_view(),'single_product'),
```
Comment (yasser-sab) [inline ecommerce/urls.py]: you forget the name here also

File: ecommerce/urls.py [hunk 3]
```diff
@@ -3,5 +3,10 @@
...
+    path('contact_us/', views.Contact.as_view(),'contact_us'),
```
Comment (yasser-sab) [inline ecommerce/urls.py]: you forget the name here also

File: ecommerce/urls.py [hunk 4]
```diff
@@ -3,5 +3,10 @@
...
+    path('about_us/', views.About.as_view(),'about_us'),
```
Comment (yasser-sab) [inline ecommerce/urls.py]: you forget the name here also

File: ecommerce/views.py [hunk 5]
```diff
@@ -8,6 +8,23 @@ class Home(View):
     def get(self,response):
         return render(response,'index.html')
 
+
+class Products(View):
+    def get(self,response):
+        return render(response, 'products.html')
+
+class SingleProduct(View):
+    def get(self,response)
```
Comment (yasser-sab) [inline ecommerce/views.py]: you should add double dots at the end of get method

File: ecommerce/views.py [hunk 6]
```diff
@@ -8,6 +8,23 @@ class Home(View):
...
+        return render(response, 'single-product.html')
+
+class Contact(View):
+     def get(self,response):
+        return render(request, 'contact.html')
+
+class About(View):
+     def get(self,response)
```
Comment (yasser-sab) [inline ecommerce/views.py]: you should add double dots at the end of get method

File: ecommerce/views.py [hunk 7]
```diff
@@ -8,6 +8,23 @@ class Home(View):
...
+    def get(self,response):
+        return render(response, 'products.html')
+
+class SingleProduct(View):
+    def get(self,response)
+        return render(response, 'single-product.html')
+
+class Contact(View):
+     def get(self,response):
+        return render(request, 'contact.html')
```
Comment (yasser-sab) [inline ecommerce/views.py]: first argument of render must be response

File: ecommerce/views.py [hunk 8]
```diff
@@ -8,6 +8,23 @@ class Home(View):
...
+
+class About(View):
+     def get(self,response)
+        return render(request, 'about.html')
```
Comment (yasser-sab) [inline ecommerce/views.py]: first argument of render must be response

File: ecommerce/views.py (hunk 6 above)
Comment (yasser-sab) [inline ecommerce/views.py]: you should add double dots at the end of get method

File: ecommerce/views.py (hunk 5 above)
Comment (yasser-sab) [inline ecommerce/views.py]: you should add double dots at the end of get method

File: ecommerce/templates/master.html [hunk 9]
```diff
@@ -52,8 +54,8 @@
                                     <li><a href="{% url 'categorie' 'accessories' %}">Accessories</a></li>
                                 </ul>
                             </li>
-                            <li><a href="contact.html">Contact Us</a></li>
-                            <li><a href="about.html">About Us</a></li>
+                            <li><a href="/contact_us">Contact Us</a></li>
```
Comment (yasser-sab) [inline ecommerce/templates/master.html]: you should add the name of the actual url specified in ecommerce/urls.py

File: ecommerce/templates/master.html [hunk 10]
```diff
@@ -52,8 +54,8 @@
...
+                            <li><a href="/about_us">About Us</a></li>
```
Comment (yasser-sab) [inline ecommerce/templates/master.html]: you should add the name of the actual url specified in ecommerce/urls.py

**Review by yasser-sab — CHANGES_REQUESTED** [review]

File: ecommerce/urls.py (hunk 1 above)
Comment (Anas-Ayoub) [reply ecommerce/urls.py]: Solved!

File: ecommerce/urls.py (hunk 2 above)
Comment (Anas-Ayoub) [reply ecommerce/urls.py]: Solved!

File: ecommerce/urls.py (hunk 3 above)
Comment (Anas-Ayoub) [reply ecommerce/urls.py]: Solved!

File: ecommerce/urls.py (hunk 4 above)
Comment (Anas-Ayoub) [reply ecommerce/urls.py]: Solved!

File: ecommerce/views.py (hunk 5 above)
Comment (Anas-Ayoub) [reply ecommerce/views.py]: Solved!

File: ecommerce/views.py (hunk 8 above)
Comment (Anas-Ayoub) [reply ecommerce/views.py]: Solved!

File: ecommerce/views.py (hunk 6 above)
Comment (Anas-Ayoub) [reply ecommerce/views.py]: Solved!

File: ecommerce/views.py (hunk 5 above)
Comment (Anas-Ayoub) [reply ecommerce/views.py]: Solved!

File: ecommerce/templates/master.html (hunk 9 above)
Comment (Anas-Ayoub) [reply ecommerce/templates/master.html]: Solved!

File: ecommerce/templates/master.html (hunk 10 above)
Comment (Anas-Ayoub) [reply ecommerce/templates/master.html]: Solved!

**Pull request merged**